CHARTER.md

```diff
@@ -0,0 +1,238 @@
+# JSON Schema Org Charter
+<!-- This document is managed in the json-schema-org/community GitHub repository. Please do NOT modify this file in another repository as changes may be overridden. -->
+
+## 0: Guiding Principles
+The JSON Schema project is part of the OpenJS Foundation which operates transparently, openly, collaboratively, and ethically. We strive to be open and transparent as much as is possible, and wish to enable anyone to interact and engage with any area of our work.
+
+Having no structure in place usually leads to one that is informal and undocumented, making it difficult to meet our own expectations of how we wish to operate. As such, we define the following charter which includes aspects of the governance model to which we subscribe and by which we operate.
+
+## 1: Scope
+JSON Schema aims to enable the confident and reliable use of the JSON data format. It does this primarily by providing specification documents which define a declarative language that allows annotation and validation of JSON documents.
+While JSON Schema's primary target is constraint-based data validation, it continues to be used across the whole stack, in different stages of a given project, and for purposes beyond its original design. We aim to enable these additional and emergent use cases.
+
+### 1.1: In-scope
+The scope of the JSON Schema project is split into two sections: primary and secondary concerns.
+Primary concerns are areas we wish to give focus to, while secondary concerns are more likely to require the creation of a Special Interest Group.
+
+Primary Concerns
+- Publication of the JSON Schema standard
+  - Validation of JSON data
+  - Semantic annotation of JSON data
+  - Extensibility
+- Critical tooling
+  - Creation of new
+  - Ingestion of existing
```

Comment:
Interesting. Are there any other examples of this in other projects?

There is at least one project, which is mostly Python, so non-JS, but is critical to JSON Schema long term, for example.

Reply:
For the sake of linking things, I raised an issue: https://github.com/openjs-foundation/cross-project-council/issues/1059

Reply:
There isn't ground for blocking the charter on this topic because multiple projects have been doing this for ages: Node.js, Fastify, and possibly many other organizations.

These are the guidelines for transferring of a repository inside the Node.js organization in https://github.com/nodejs/admin/blob/main/transfer-repo-into-the-org.md.

Note that I recommend the two bullet points under "critical tooling" to be removed.

Reply:
I've removed the three lines under "ciritical tooling", for now.